<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1220</width>
    <height>801</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Menu">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>70</width>
      <height>801</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
		
	background-color: qlineargradient(spread:pad, x1:0.49, y1:0.0164091, x2:0.495, y2:1, stop:0 rgba(4, 2, 14, 255), stop:1 rgba(0, 20, 77, 255));
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="HomeButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>69</width>
      <height>69</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	border: 0px solid;
	background:transparent;
}

QPushButton:hover
{
	border: 0px solid;
	background-color: rgb(23, 25, 71);
}

QPushButton:pressed
{
	border: 0px solid;
	
	background-color: rgb(20, 21, 44);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="OpenButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>90</y>
      <width>69</width>
      <height>69</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	border: 0px solid;
	background:transparent;
}

QPushButton:hover
{
	border: 0px solid;
	background-color: rgb(23, 25, 71);
}

QPushButton:pressed
{
	border: 0px solid;
	
	background-color: rgb(20, 21, 44);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="AddButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>160</y>
      <width>69</width>
      <height>69</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	border: 0px solid;
	background:transparent;
}

QPushButton:hover
{
	border: 0px solid;
	background-color: rgb(23, 25, 71);
}

QPushButton:pressed
{
	border: 0px solid;
	
	background-color: rgb(20, 21, 44);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="EditButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>230</y>
      <width>69</width>
      <height>69</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	border: 0px solid;
	background:transparent;
}

QPushButton:hover
{
	border: 0px solid;
	background-color: rgb(23, 25, 71);
}

QPushButton:pressed
{
	border: 0px solid;
	
	background-color: rgb(20, 21, 44);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="SaveButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>300</y>
      <width>69</width>
      <height>69</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	border: 0px solid;
	background:transparent;
}

QPushButton:hover
{
	border: 0px solid;
	background-color: rgb(23, 25, 71);
}

QPushButton:pressed
{
	border: 0px solid;
	
	background-color: rgb(20, 21, 44);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="PreferencesButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>730</y>
      <width>69</width>
      <height>69</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	border: 0px solid;
	background:transparent;
}

QPushButton:hover
{
	border: 0px solid;
	background-color: rgb(23, 25, 71);
}

QPushButton:pressed
{
	border: 0px solid;
	
	background-color: rgb(20, 21, 44);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="InfoLabel">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>0</y>
      <width>1151</width>
      <height>220</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(218, 218, 218);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="StructLabel">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>220</y>
      <width>1151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:reflect, x1:0, y1:0.534, x2:0.504615, y2:0.523, stop:0 rgba(70, 78, 125, 255), stop:1 rgba(81, 86, 125, 255));</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="TabelLabel">
    <property name="geometry">
     <rect>
      <x>69</x>
      <y>281</y>
      <width>1151</width>
      <height>521</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0.322692, y1:0.597, x2:0, y2:1, stop:0 rgba(2, 32, 70, 255), stop:1 rgba(43, 43, 89, 255));</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>300</y>
      <width>1101</width>
      <height>491</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto Mono</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(48, 72, 121);
border: 1px solid white;
gridline-color: white;
color: #DADADA;
selection-background-color: rgb(66, 99, 166);</string>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="gridStyle">
     <enum>Qt::DotLine</enum>
    </property>
   </widget>
   <widget class="QFrame" name="EmptyFrame">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>330</y>
      <width>1041</width>
      <height>331</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="noBooks">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>20</y>
       <width>561</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Roboto Mono</family>
       <pointsize>30</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white;</string>
     </property>
     <property name="text">
      <string>There are no books</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="noBooks_2">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>180</y>
       <width>451</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Roboto Mono</family>
       <pointsize>30</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white;</string>
     </property>
     <property name="text">
      <string>Add new book or</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="noBooks_3">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>180</y>
       <width>161</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Roboto Mono</family>
       <pointsize>30</pointsize>
       <weight>75</weight>
       <bold>true</bold>
       <underline>true</underline>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #4571E3;</string>
     </property>
     <property name="text">
      <string>open</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="noBooks_4">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>180</y>
       <width>251</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Roboto Mono</family>
       <pointsize>30</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white;</string>
     </property>
     <property name="text">
      <string>the file</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="EmptyOpen">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>190</y>
       <width>121</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:transparent;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="AuthorButton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>220</y>
      <width>311</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto Mono</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	border: 0px solid;
	background:transparent;
	color: rgb(159, 159, 159);
}

QPushButton:hover
{
	border: 0px solid;
	background-color: #5C669E;
	color: rgb(159, 159, 159);
}

QPushButton:pressed
{
	border: 0px solid;
	background-color: rgb(83, 91, 135);
	color: rgb(159, 159, 159);
}</string>
    </property>
    <property name="text">
     <string>Author</string>
    </property>
   </widget>
   <widget class="QPushButton" name="NameButton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>220</y>
      <width>361</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto Mono</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	border: 0px solid;
	background:transparent;
	color: rgb(159, 159, 159);
}

QPushButton:hover
{
	border: 0px solid;
	background-color: #5C669E;
	color: rgb(159, 159, 159);
}

QPushButton:pressed
{
	border: 0px solid;
	background-color: rgb(83, 91, 135);
	color: rgb(159, 159, 159);
}</string>
    </property>
    <property name="text">
     <string>Name</string>
    </property>
   </widget>
   <widget class="QPushButton" name="YearButton">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>220</y>
      <width>145</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto Mono</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	border: 0px solid;
	background:transparent;
	color: rgb(159, 159, 159);
}

QPushButton:hover
{
	border: 0px solid;
	background-color: #5C669E;
	color: rgb(159, 159, 159);
}

QPushButton:pressed
{
	border: 0px solid;
	background-color: rgb(83, 91, 135);
	color: rgb(159, 159, 159);
}</string>
    </property>
    <property name="text">
     <string>Year</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PagesButton">
    <property name="geometry">
     <rect>
      <x>905</x>
      <y>220</y>
      <width>145</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto Mono</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	border: 0px solid;
	background:transparent;
	color: rgb(159, 159, 159);
}

QPushButton:hover
{
	border: 0px solid;
	background-color: #5C669E;
	color: rgb(159, 159, 159);
}

QPushButton:pressed
{
	border: 0px solid;
	background-color: rgb(83, 91, 135);
	color: rgb(159, 159, 159);
}</string>
    </property>
    <property name="text">
     <string>Pages</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PriceButton">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>220</y>
      <width>145</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto Mono</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	border: 0px solid;
	background:transparent;
	color: rgb(159, 159, 159);
}

QPushButton:hover
{
	border: 0px solid;
	background-color: #5C669E;
	color: rgb(159, 159, 159);
}

QPushButton:pressed
{
	border: 0px solid;
	background-color: rgb(83, 91, 135);
	color: rgb(159, 159, 159);
}</string>
    </property>
    <property name="text">
     <string>Price</string>
    </property>
   </widget>
   <widget class="QLabel" name="image">
    <property name="geometry">
     <rect>
      <x>151</x>
      <y>20</y>
      <width>151</width>
      <height>182</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="AuthorInfo">
    <property name="geometry">
     <rect>
      <x>321</x>
      <y>30</y>
      <width>381</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto Mono</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="BookInfo">
    <property name="geometry">
     <rect>
      <x>321</x>
      <y>80</y>
      <width>501</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sylfaen</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="LangInfo">
    <property name="geometry">
     <rect>
      <x>321</x>
      <y>140</y>
      <width>381</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sylfaen</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="PlayButton">
    <property name="geometry">
     <rect>
      <x>841</x>
      <y>30</y>
      <width>351</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: rgb(79, 207, 28);
	color: rgb(255, 255, 255);
	border-radius: 20px;
}
QPushButton:hover
{
	background-color: rgb(94, 238, 32);
	color: rgb(255, 255, 255);
	border-radius: 20px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="BooksButton">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>30</y>
      <width>351</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: rgb(41, 109, 197);
	color: rgb(255, 255, 255);
	border-radius: 20px;
}
QPushButton:hover
{
	background-color: rgb(51, 120, 210);
	color: rgb(255, 255, 255);
	border-radius: 20px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="DeleteButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>390</y>
      <width>69</width>
      <height>69</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	border: 0px solid;
	background:transparent;
}

QPushButton:hover
{
	border: 0px solid;
	background-color: rgb(23, 25, 71);
}

QPushButton:pressed
{
	border: 0px solid;
	
	background-color: rgb(20, 21, 44);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QFrame" name="WelcomeFrame">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>0</y>
      <width>1131</width>
      <height>221</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="MainIcon">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>0</y>
       <width>209</width>
       <height>196</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="Created">
     <property name="geometry">
      <rect>
       <x>970</x>
       <y>180</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #626262;</string>
     </property>
     <property name="text">
      <string>Created by senoron</string>
     </property>
    </widget>
    <widget class="QLabel" name="Virtual">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>291</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Roboto Mono</family>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>VirtualLibrary</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>79</y>
      <width>58</width>
      <height>2</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>7</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(208, 208, 208);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>379</y>
      <width>58</width>
      <height>2</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>7</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(208, 208, 208);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>InfoLabel</zorder>
   <zorder>Menu</zorder>
   <zorder>HomeButton</zorder>
   <zorder>OpenButton</zorder>
   <zorder>AddButton</zorder>
   <zorder>EditButton</zorder>
   <zorder>SaveButton</zorder>
   <zorder>PreferencesButton</zorder>
   <zorder>StructLabel</zorder>
   <zorder>TabelLabel</zorder>
   <zorder>tableWidget</zorder>
   <zorder>EmptyFrame</zorder>
   <zorder>AuthorButton</zorder>
   <zorder>NameButton</zorder>
   <zorder>YearButton</zorder>
   <zorder>PagesButton</zorder>
   <zorder>PriceButton</zorder>
   <zorder>image</zorder>
   <zorder>AuthorInfo</zorder>
   <zorder>BookInfo</zorder>
   <zorder>LangInfo</zorder>
   <zorder>PlayButton</zorder>
   <zorder>BooksButton</zorder>
   <zorder>DeleteButton</zorder>
   <zorder>WelcomeFrame</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
